### DISCUSSION TITLE
Connectivity issues with g7 and Samsung S23

### DISCUSSION THREAD

**Discussion by joshg1994** [NightscoutFoundation/xDrip#3373] [Q&A]
Hello, I've noticed often there are Bluetooth connectivity issues where It may take 10-20 mins to reconnect. I am using the g7 with my Samsung s23. Any tips or reasons why I'm having frequent sensor delays? 
![Screenshot_20240229_173735_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/157648698/3f743450-eefc-46f2-8dd5-c232803ac310)

**Comment by Navid200** [comment]
I suggest that you update to our alpha release by downloading and installing the following over your current xDrip.  So, there is no need to uninstall first.
https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.01.16

Considering you use a Samsung, 
have you disabled trust auto connect?
have you enabled special paring workaround?

**Reply by joshg1994** [reply]
I have enabled the special pairing, and downloaded the update, thank you.

**Reply by joshg1994** [reply]
A couple days since following your recommendations and still having delays. Are there areas that you would/wouldn't recommend placing the sensor? 
![Screenshot_20240302_114208_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/157648698/63b622ac-58f9-4382-85fe-477928211b26)

**Reply by Navid200** [reply]
How is the sensor placement related?  Does this happen only when you are sleeping?
I personally place my sensor on my arm.

**Reply by joshg1994** [reply]
It's currently placed on my upper glute below my wait band. So I'm wondering if when im sitting, this restricts the connectivity, which is when it occurs a lot of the times. But wtf do I know, I'm not the developer. Really appreciate the smart ass response.

**Reply by Navid200** [reply]
It was not meant to be a smart anything response.  Humor in text is a mistake because it can cause misunderstanding.
You have a connectivity problem and if the problem only happens when you are sleeping, the placement could be the cause because Bluetooth range decreases dramatically in water and our body is mostly water.  
 
You can experiment with sensor placement next time to see if it makes a difference.
